docs/satellite-clone.md

```diff
@@ -3,17 +3,18 @@
  **Important Notes:**
 
   - The playbook will update the cloned Satellite hostname to match the hostname of the original Satellite from which the backup is generated.
-  - DHCP, DNS, and TFTP will be disabled during the install to avoid configuration errors. If you want to use provisioning on the cloned Satellite, you will have to manually re-enable these settings.
+  - DHCP, DNS, and TFTP will be disabled during the install to avoid configuration errors. If you want to use provisioning on the cloned Satellite, you will have to manually re-enable these settings after the playbook run.
   - The playbook will reset the admin password to "changeme".
-  - The playbook installs Satellite and may may take a while to complete.
-  - Make sure that the destination node has the same OS version as the original Satellite from which the backup data was generated.
+  - The playbook run may may take a while to complete.
+  - The destination node must have the same OS version as the original Satellite from which the backup data was generated.
+    *Note:* You can optionally choose to clone a rhel6 Satellite server to a rhel7 machine.  In this case, you must update the variable `rhel_migration` to true as explained later in this document.
```

Comment:
I think we can say here that a RHEL 7 migration is supported on Satellite 6.2 only (if that is what we are supporting)